rails/Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+gem 'benchmark-ips', github: 'kirs/benchmark-ips', branch: 'quiet-mode'
+gem 'rails', path: '/rails'
+gem 'ffaker'
+gem 'sqlite3'
+gem 'stackprof'
+gem 'mysql2'
+gem 'pg'
+gem 'arel', github: 'rails/arel', branch: 'master'
```

Comment:
For Gems (ActiveRecord) that may possibly affect the benchmarks, do you think we should lock them to specific versions? Otherwise, we may end up benchmarking a moving target.